app/src/app/list/list.component.html

```diff
@@ -1,9 +1,27 @@
 <app-server-error [error]="error$ | async"></app-server-error>
-<div class="poll-list" *ngIf="(publicList$ | async)?.length">
-  <h3>Polls you participate in</h3>
-  <app-polls-table [polls]="publicList$ | async"></app-polls-table>
+<div id="mat-tab-container">
+  <mat-tab-group mat-align-tabs="center">
+    <mat-tab  label="Participating">
```

Comment:
- The tab should not be displayed if there is no poll in the list.
- Moreover, I propose "Participate" instead of "Participating".
- There is too much space between the title bar and the tabs. Since tabs are a bit special, compared to other pages, we could use negative margin here.